virt/monitoring/virt-exposing-downward-metrics.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-exposing-downward-metrics"]
+= Exposing downward metrics for virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-exposing-downward-metrics
+
+toc::[]
+
+As an administrator, you can expose a limited set of host and virtual machine (VM) metrics to a guest VM by configuring a YAML file. The YAML file must include a `virtio-serial` port, a `downwardMetrics` device, and a `DownwardMetrics` feature gate. Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.
```

Comment:
```suggestion
As an administrator, you can expose a limited set of host and virtual machine (VM) metrics to a guest VM by configuring a YAML file. The YAML file must include a `downwardMetrics` device, and the `DownwardMetrics` feature gate has to be enabled. Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.
```

The sentence `Enabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.`, IMHO, may lead to users to believe that if the feature gate is enabled, a restricted view is presented, which is not true. If the feature gate is not enabled, the user will not able to get any metric (in fact, the user will not be able to create a VM requesting downwardMetrics at all). If enabled, the user will be able to see all downwardMetrics exposed.

The idea behind the feature gate is to help the admin to restrict if VMs will be able or not to pull the downwardMetrics. Therfore, I will rephrase as: 

`Disabling the feature gate helps to ensure additional protection in restricting viewing of the metrics results.`